README.md

```diff
@@ -52,14 +52,31 @@ Eser 'Laroux' Özvataf
 
 - Çalışan adayının özgeçmişine "göz atmak" yerine özenle okuyup değerlendirecek, özgeçmişle ilgili anlamadığı, detaylı bilmek istediği ve eksik olduğunu düşündüğü noktaları sorarak ilgili pozisyon için adayın artı ve eksilerini değerlendirebilecek,
 
-- Maaşı kişinin yaşam şartları (yaşı, evli olması, çocuk sahibi olması, aracı olması, ev kirası, borçları) üzerinden değil de yapacağı iş üzerinden belirleyecek,
+- Maaşı kişinin yaşam şartları (yaşı, evli olması, çocuk sahibi olması, aracı olması, ev kirası, borçları) üzerinden değil de yapacağı iş üzerinden ve kişinin hayatı boyunca edindiği tebrübelere göre belirleyecek,
```

Comment:
"ve kişinin sahip olduğu tecrübe ile doğru orantılı" şeklinde değiştirerek bunu ekleyelim derim :)